src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormattingPassBase.cs

```diff
@@ -167,7 +167,7 @@ protected async Task<List<TextChange>> AdjustIndentationAsync(FormattingContext
             var line = context.SourceText.Lines[i];
             var lineStart = line.GetFirstNonWhitespacePosition() ?? line.Start;
             var lineStartSpan = new TextSpan(lineStart, 0);
-            if (!ShouldFormat(context, lineStartSpan, allowImplicitStatements: true))
+            if (!ShouldFormatLine(context, lineStartSpan, allowImplicitStatements: true))
```

Comment:
This ShouldFormat method is arguably still doing too much, but this particular usage has annoyed me before, so I done fixed it.